src/appleseed.python/bindaov.cpp

```diff
@@ -103,7 +121,8 @@ void bind_aov()
         .def("get_channel_count", &AOV::get_channel_count)
         .def("get_channel_names", &get_channel_names)
         .def("has_color_data", &AOV::has_color_data)
-        .def("get_image", &AOV::get_image, bpy::return_value_policy<bpy::reference_existing_object>());
+        .def("get_image", &AOV::get_image, bpy::return_value_policy<bpy::reference_existing_object>())
+        .def("get_crypto_image", &get_crypto_image, bpy::return_value_policy<bpy::reference_existing_object>());
```

Comment:
Can you rename "crypto" to "cryptomatte" everywhere please? It disturbs me 😂

Reply:
Oh okay